### ISSUE TITLE
When NVDA is on, for selection does not go beyond the optionLabel, when it is displayed as selected

### ISSUE THREAD

**Issue by veselints** [telerik/kendo-ui-core#6706]
### Bug report
When NVDA is on, for selection does not go beyond the optionLabel, when it is displayed as selected

### Reproduction of the problem
* Turn on NVDA
* Open the following demo in Firefox: https://demos.telerik.com/kendo-ui/dropdownlist/cascadingdropdownlist
* Using the Tab key focus the first DDL
* Press Alt+Down Arrow to open the widget popup
* Press the Down Arrow - the focus and selection does not move to the next option in the list

### Expected/desired behavior
The focus and selection should move beyond the optionLabel item in the list

### Environment
* **Kendo UI version:** 2022.1.119
* **Browser:** [Firefox,Chrome]

**Comment by veselints** [comment]
https://github.com/telerik/kendo-ui-core/issues/3789

**Comment by veselints** [comment]
Depends on:
https://github.com/telerik/kendo/issues/14610

**Comment by veselints** [comment]
- [ ] Upon focus, NVDA does not announce selected item;
- [ ] NVDA in Firefox announces twice each opinion upon navigation when popup is open;
